runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -113,6 +121,15 @@
 	private final XpClient xpClient = new XpClient();
 	private final XpState xpState = new XpState();
 	private final XpPauseState xpPauseState = new XpPauseState();
+	private Map<String, Skill> skillMap = new HashMap<String, Skill>()
```

Comment:
1. Put this in the Skill class
2. It should be `private static final`
3. It should also be an `ImmutableMap`